modules/statics/src/account.ts

```diff
@@ -331,22 +331,6 @@ export class AdaCoin extends AccountCoinToken {
   }
 }
 
-/**
- * The Trx network supports tokens
- * Trx tokens are identified by their smart contract address
- *
- */
-export class TrxCoin extends AccountCoinToken {
```

Comment:
Unrecorded breaking change in commit message.

Reply:
You will need to break this PR into two commits. One for the breaking changes in statics to not bump sdk-coin-trx to the wrong version.